### PR TITLE
Change request/#5520 order details form

### PR DESCRIPTION
**Pull request by katia9261**
Changes for [change request](https://github.com/ita-social-projects/GreenCity/issues/5520) for order details form

### CODE REVIEW

File: src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html [hunk 1]
```diff
@@ -49,6 +49,7 @@ <h3 class="form__headline">{{ 'order-details.title' | translate }}</h3>
                 min="0"
                 max="999"
                 formControlName="actualQuantity{{ bag.id }}"
+                [attr.disabled]="isOrderCancelled && isOrderPaid ? true : null"
```
Comment (KKatiaa) [inline src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html]: You should use readonly instead attr.disabled

**Review by KKatiaa — APPROVED** [review]

File: src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html (hunk 1 above)
Comment (katia9261) [reply src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html]: Thanks, I have changed it to just `disabled`)

File: src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.ts [hunk 2]
```diff
@@ -198,14 +216,17 @@ export class UbsAdminOrderDetailsFormComponent implements OnInit, OnChanges {
...
     priceWithoutCertificate = Math.max(priceWithoutCertificate, 0);
 
-    this.overpayment = this.orderDetails.bonuses + this.orderDetails.paidAmount - priceWithoutCertificate - this.writeoffAtStationSum;
+    const baseSumOfOrder = this.orderDetails.bonuses + this.orderDetails.paidAmount + this.orderDetails.certificateDiscount;
 
-    if (this.showUbsCourier) {
-      this.overpayment += this.courierPrice * (this.isOrderCancelled ? -1 : 1);
+    if (this.isOrderBroughtByHimself) {
+      this.overpayment = baseSumOfOrder - this.writeoffAtStationSum;
+    } else if (this.showUbsCourier) {
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.ts:227]: why you use 'else if' here?

**Review by hnativlyubomyr — APPROVED** [review]

File: src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.ts (hunk 2 above)
Comment (katia9261) [reply src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.ts:227]: 'else if' in this case allows to handle multiple scenarios for calculating the overpayment based on [different conditions](https://github.com/ita-social-projects/GreenCity/issues/5520).

**Review by atsa21 — APPROVED** [review]

**Review by AlexW8719 — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Review by ruslan-teh — APPROVED** [review]

**Pull request merged**